extensions/wikia/ARecoveryEngine/ARecoveryEngineApiController.class.php

```diff
@@ -0,0 +1,21 @@
+<?php
+class ARecoveryEngineApiController extends WikiaController {
+
+	const DEFAULT_TEMPLATE_ENGINE = WikiaResponse::TEMPLATE_ENGINE_MUSTACHE;
+
+	public function getDelivery() {
+		$resourceLoader = new ResourceLoaderAdEngineSourcePointCSDelivery();
+		$resourceLoaderContext = new ResourceLoaderContext( new ResourceLoader(), new FauxRequest() );
+		$source = $resourceLoader->getScript( $resourceLoaderContext );
+		$this->response->setContentType( 'text/javascript; charset=utf-8' );
+		$this->response->setVal( 'code', $source );
```

Comment:
I found it! According to https://github.com/Wikia/app/blob/dev/includes/wikia/nirvana/WikiaDispatchableObject.class.php#L150
Can't we use `setBody` with `$source` and get rid of `ARecoveryEngineApiController_getdelivery.mustache`?